src/LLMProviders/chainManager.ts

```diff
@@ -317,12 +317,26 @@ export default class ChainManager {
     this.validateChatModel();
     this.validateChainInitialization();
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const chatModel = (ChainManager.chain as any).last.bound;
```

Comment:
What's the difference between using this vs. "ChatModelManager.getChatModel()"?

Reply:
@logancyang  At first, I plan to use `ChatModelManager.getChatModel()`, but I copied your code for consistency.